pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -165,17 +165,31 @@ func convertVirtualService(r *KubernetesResources, gatewayMap map[parentKey]map[
 		}
 	}
 
+	// sort HTTPRoutes by creation timestamp and namespace/name
```

Comment:
`sortConfigByCreationTime` uses `name.namespace` but here I'm using `namespace/name`, I'm not sure if changing the order would break something, if it's ok I will change it